doc/leaderf.txt

```diff
@@ -830,6 +830,20 @@ g:Lf_JumpToExistingWindow                       *g:Lf_JumpToExistingWindow*
 
     Default value is 0.
 
+g:Lf_ShowDevIcons                               *g:Lf_ShowDevIcons*
+    Use vim-devicons to display icons.
+    NOTE: https://github.com/ryanoasis/vim-devicons is required.
+    
+    Support commands: buffer,file,mru,window
+
+    Default value is 1.
+
+g:Lf_DevIconsFont                               *g:Lf_DevIconsFont*
+    Specifies the font used to display the vim-devicons icon.
+    If it is empty, use the default font.
```

Comment:
The font is only available for gui vim.

Reply:
I didn't know that.I'm sorry.
I will remove `g:Lf_DevIconsFont` so that it can be configured with `Lf_hl_devIcons`.